addon/templates/components/sl-input.hbs

```diff
@@ -13,6 +13,7 @@
 {{/if}}
 
 {{input
+    id=inputId
```

Comment:
The input within the `sl-radio` component doesn't need an id assigned because the label wraps the input, hence a for attribute it not required on the label.

`sl-select` component hides the original input provided in the template and creates it's own input within a div. I was able to assign the id after the element renders in this code https://github.com/softlayer/sl-ember-components/pull/798/files#diff-6e21ecc6b00cadbe559301f6b81a8b3eR280.